permissions/models.py

```diff
@@ -56,7 +56,7 @@
     ),
     (
         "can_view_reservations",
-        _("Can create, edit and delete equipment in the whole system"),
+        _("Can view equipments in the whole system"),
```

Comment:
"Can view reservations in the whole system"

Reply:
Fixed 🤦